index.bs

```diff
@@ -313,6 +313,8 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
 
     - The track shall be a valid [=MIAF image sequence=].
     - The track handler for an [=AV1 Image Sequence=] shall be <code>pict</code>.
+    - The track shall have only one sample description entry.
+    - If multiple [=Sequence Header OBUs=] are present in the track payload, they shall be identical.
```

Comment:
The exact requirement in the AV1 specification (Section 7.5, page 205) is the following:

> Within a particular coded video sequence, the contents of sequence_header_obu must be bit-identical each time the sequence header appears except for the contents of operating_parameters_info.

To match the AV1 specification exactly, we can change "identical" to "bit-identical" and add "except for the contents of operating_parameters_info" to the end of this sentence. But I am not sure if we need to be that precise.